### PR TITLE
Feedforward examples

### PR DESCRIPTION
**Pull request by jrobble**
This change is [Reviewable](https://reviewable.io/reviews/openmpf/openmpf.github.io/52)

### CODE REVIEW

File: docs/docs/Feed-Forward-Guide.md [hunk 1]
```diff
@@ -56,16 +57,33 @@ When `FEED_FORWARD_TOP_CONFIDENCE_COUNT` is set to a number greater than 0, say
...
+
+The major advantage of using a superset region is constant size. Some algorithms require the search space in each frame to be a constant size in order to successfully track objects.
 
-The major advantage is that the superset region is a constant size. Some algorithms require the search space to be a constant size in order to successfully track objects.
+A disadvantage is that the superset region will often be larger than any specific detection region, so the search space is not restricted to the smallest possible size in each frame; however, in many cases the search space will be significantly smaller than the whole frame.
 
-A disadvantage is that the superset region will be larger than any specific detection region, so the search space is not restricted to the smallest possible size, but in many cases the search space will be significantly reduced compared to the whole frame.
+For example, in the worst case, a feed forward track may capture a person moving from the upper left corner of a video to the lower right corner. In that case the superset region will be the entire width and height of the frame, so `SUPERSET_REGION` devolves into `FRAME`.
```
Comment (clnowacki) [inline docs/docs/Feed-Forward-Guide.md]: I would reword this as

"In the worst case, a feed forward track might, for example, capture a person moving..."

File: docs/docs/Feed-Forward-Guide.md [hunk 2]
```diff
@@ -56,16 +57,33 @@ When `FEED_FORWARD_TOP_CONFIDENCE_COUNT` is set to a number greater than 0, say
...
 
-For example, in the worst case, a feed forward track may capture a person moving from the lower right corner of a video to the upper left corner. In that case the superset region will be the entire width and height of the frame, so `SUPERSET_REGION` devolves into `FRAME`.
+For example, in the average case, a feed forward track may capture a person moving in the upper left quadrant of a video. In that case `SUPERSET_REGION` is able to filter out 75% of the rest of the frame data. In the example shown in the above diagram, `SUPERSET_REGION` is able to filter out 83% of the rest of the frame data.
```
Comment (clnowacki) [inline docs/docs/Feed-Forward-Guide.md]: I would reword this as 

"In a more typical case, a feed forward track might capture..."

I don't like calling it "average" which implies some numerical property.

**Comment by jrobble** [comment]
Review status: 0 of 10 files reviewed at latest revision, 2 unresolved discussions.

---

*[docs/docs/Feed-Forward-Guide.md, line 74 at r1](https://reviewable.io:443/reviews/openmpf/openmpf.github.io/52#-L5pNBRs96Roc-MjbffH-r1-74:-L5tbqFAQqb9hLXPF0om:b-896fix) ([raw file](https://github.com/openmpf/openmpf.github.io/blob/c027f0a7423fe33f42219c5f9ed11f79fc467587/docs/docs/Feed-Forward-Guide.md#L74)):*
<details><summary><i>Previously, clnowacki wrote…</i></summary><blockquote>

I would reword this as

"In the worst case, a feed forward track might, for example, capture a person moving..."
</blockquote></details>

Done.

---

*[docs/docs/Feed-Forward-Guide.md, line 76 at r1](https://reviewable.io:443/reviews/openmpf/openmpf.github.io/52#-L5pNBRs96Roc-MjbffH-r1-76:-L5tcBEi6q3Jc-EWi5V0:b-896fix) ([raw file](https://github.com/openmpf/openmpf.github.io/blob/c027f0a7423fe33f42219c5f9ed11f79fc467587/docs/docs/Feed-Forward-Guide.md#L76)):*
<details><summary><i>Previously, clnowacki wrote…</i></summary><blockquote>

I would reword this as 

"In a more typical case, a feed forward track might capture..."

I don't like calling it "average" which implies some numerical property.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/openmpf/openmpf.github.io/52)*
